.gitignore

```diff
@@ -135,4 +135,5 @@ dist
 .yarn/install-state.gz
 .pnp.*
 
-zztemp
\ No newline at end of file
+zztemp
+modules
```

Comment:
not a problem for me but I'm curious why gitignoring `modules`? As for my zztemp, it's because it's easier to locate that directory in editors when it's alphabetically placed near the bottom.

Reply:
`module` is a new thing that is coming to remult (in 3.0.6 probably).

To get an idea of it, here is the API:
![image](https://github.com/user-attachments/assets/b4682bd8-6628-4ed7-b3ad-6ca44452feee)